src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -186,6 +187,31 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
                     // TO DO -> call to reinitialize the database.
                 }
             });
+
+        // Subscription to vulnerability detector translations update events.
+        m_translationsSubscription = std::make_unique<TRouterSubscriber>(
+            updaterPolicy.at("translationsTopic"), "vulnerability_feed_manager", isLocalSubscriber);
```

Comment:
The router topic name could be in a constant. So subscribers and publishers could use the same constant to eliminate the possibility of being different.